contract/src/postal-service.contract.js

```diff
@@ -12,16 +12,18 @@ export const meta = harden({
 // compatibility with an earlier contract metadata API
 export const { customTermsShape } = meta;
 
+let issuerNumber = 1;
```

Comment:
move this inside `start()`, please. No static mutable state.

Reply:
I considered making it durable, but saw that this was what the swap contract does already ,so I figured it's not critical to preserve the issuer numbers. Is no static mutable state a hard rule? I can update it in both contracts if needed.

Reply:
> I can update it in both contracts if needed.

Please do.

> Is no static mutable state a hard rule?

It's part of [OCap Discipline](https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline) that isn't (yet?) enforced by our platform.

[SES lockdown docs](https://github.com/endojs/endo/tree/master/packages/ses#lockdown) say:

> SES introduces the `lockdown()` function. Calling `lockdown()` alters the surrounding execution environment, or **realm**, such that no two programs running in the same realm can observe or affect each other until they have been introduced, and even then can only interact through their own exposed interfaces.

But that's not quite true: SES doesn't (yet?) get rid of module-level static mutable state such as

```js
let callCount = 0;

export const complementaryColor = _ => {
  callCount += 1;
  return 'green';
}

export const colorCallCount = () => callCount;
```

So one program could do

```js
import { complementaryColor } from './fun-stuff.js';

console.log(complementaryColor('blue');
```

and another could do

```js
import { colorCallCount } from './fun-stuff.js`;

console.log('there have been', colorCallCount(), 'calls to complementaryColor');
```


cc @kriskowal @erights

Reply:
Nothing in the postal service contract is durable, so that would be above end beyond the call of duty here.

Reply:
Ah I see what you mean, move it into the function scope but don't necessarily make it durable. Done.